upgrade-notes/upgrade_notes.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+if ($argc < 2) {
+    echo "Error: version is not set. Pass the version as argument.\n";
+    exit(1);
+}
+
+$version = $argv[1];
+$upgradeNotesPath = __DIR__;
+$layoutFile = "{$upgradeNotesPath}/_layout.md";
+$outputFile = "UPGRADE-{$version}.md";
+
+const BACKEND_NOTES_PLACEHOLDER = '<!-- backendNotes -->';
+const STOREFRONT_NOTES_PLACEHOLDER = '<!-- storefrontNotes -->';
+
+function concatenateFiles($prefix, $upgradeNotesPath) {
+    $content = "";
+    $files = glob("{$upgradeNotesPath}/{$prefix}*");
+    $lastIndex = count($files) - 1;
+
+    foreach ($files as $index => $file) {
+        if (is_file($file)) {
+            $content .= file_get_contents($file);
+            // Add an empty line after each file except the last one
+            if ($index != $lastIndex) {
```

Comment:
When the last file is not a file (is_file condition above), new two lines are added to the last item. I'm not sure how `glob()` sorts items and maybe it is not a problem 🤷‍♂️

Reply:
There are no aded lines if $file is not a file, no?

Reply:
Got it now. After our discussion i'm letting it go as it is.